QWeb/internal/browser/edge.py

```diff
@@ -79,12 +79,14 @@ def open_browser(executable_path: str = "msedgedriver",
         options.add_argument("--headless")  # pylint: disable=no-member
     if 'prefs' in kwargs:
         tmp_prefs = kwargs.get('prefs')
-        if isinstance(tmp_prefs, dict):
-            prefs = tmp_prefs
-        else:
-            prefs = util.prefs_to_dict(str(tmp_prefs).strip())
```

Comment:
Check review comment (change suggestion) to `util.validate_prefs()`

With that change this can be simplified to just
`prefs = util.prefs_to_dict(tmp_prefs)`

Reply:
not changed in this pr, so will make a specific refactor task about this

Reply:
was done due to linting actually. TBH can't see it being worth to be changed now. Maybe in later refactoring rounds.

Reply:
changed validate_prefs to parse_prefs